src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -80,22 +80,22 @@ const styles = StyleSheet.create({
     alignSelf: 'center'
   },
   tabItemText: {
-    color: colors.alto,
+    color: COLORS.alto,
     fontWeight: '500',
     fontSize: normalizeFontSize(12.5),
     paddingVertical: 10,
     textAlign: 'center'
   },
   tabItemTextFocus: {
-    color: colors.black,
+    color: COLORS.black,
     fontWeight: '500',
     fontSize: normalizeFontSize(12.5),
     lineHeight: 16.94,
     textAlign: 'center',
     paddingVertical: 10
   },
   underlineFocus: {
-    borderBottomColor: colors.bondi_blue,
+    borderBottomColor: COLORS.blue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values in the style definitions have been updated to use the new `COLORS` object. This change is consistent with the update to the import statement and should not cause any issues. However, the color for `underlineFocus` has been changed from `colors.bondi_blue` to `COLORS.blue`. If this change was intentional and `COLORS.blue` is indeed the desired color, then this change is fine. However, if `colors.bondi_blue` was the correct color, this line should be reverted.
```diff
- color: colors.alto,
+ color: COLORS.alto,

- color: colors.black,
+ color: COLORS.black,

- borderBottomColor: colors.bondi_blue,
+ borderBottomColor: COLORS.blue,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->